src/HomeComponent/HomeComponent.js

```diff
@@ -19,28 +20,31 @@ class HomeComponent extends Component {
   handleClick = () => {
     this.setState({ gif: this.state.list[Math.floor(Math.random() * this.state.list.length)]});
   }
-  
+
   componentDidMount() {
-    console.log('componentDidMount')
-    axios.get(`https://api.giphy.com/v1/gifs/search?api_key=${apikey}&q=cheeseburgers`).then((response) => {
+    // console.log('componentDidMount')
+    axios.get(`https://api.giphy.com/v1/gifs/search?api_key=${apikey}&q=cats`).then((response) => {
       console.log('response', response)
```

Comment:
Remove console logs from your code when you no longer need them, especially when merging into master. Ideally, we should have no console logs in our master branch, except when handling errors.